app/charts/shared/axis-width-linear.tsx

```diff
@@ -57,17 +56,21 @@ export const AxisWidthLinear = () => {
       });
 
       g.selectAll(".tick line")
-        .attr("stroke", gridColor)
+        .attr("y1", 10)
+        .attr("stroke", (_, i) => (i === 0 ? "#000000" : gridColor))
         .attr("stroke-width", 1);
       g.selectAll(".tick text")
         .attr("font-size", labelFontSize)
         .attr("font-family", fontFamily)
         .attr("fill", labelColor)
-        .attr("dy", labelFontSize)
+        .attr("dy", labelFontSize + 10)
         .attr("text-anchor", "middle");
-      g.select("path.domain").attr("stroke", gridColor);
+      g.select("path.domain")
+        .attr("stroke", gridColor)
+        .attr("opacity", chartType === "bar" ? 0 : 1);
```

Comment:
Super-extra-nit: We could also remove the domain if the chart is a bar, one less element 🥹😅 

```typescript
if (chartType === "bar") {
  g.select("path.domain").remove();
} else { ... }
```